src/python/espressomd/visualization_opengl.pyx

```diff
@@ -1579,7 +1579,7 @@ class openGLLive:
         # LOOK FOR LB ACTOR
         if self.specs['LB_draw_velocity_plane'] or self.specs['LB_draw_nodes'] or self.specs['LB_draw_node_boundaries']:
             for a in self.system.actors:
-                types = [types.append(espressomd.lb.LBFluid)]
+                types = [espressomd.lb.LBFluid]
```

Comment:
why is that happening in a loop?

Reply:
the new syntax in ea9dd76 is easily extensible: `if isinstance(a, (espressomd.lb.HydrodynamicInteraction, espressomd.walberla.lb.LB))`

Reply:
this is now happening outside the loop